openapi.yaml

```diff
@@ -158,162 +248,227 @@ components:
-          example: 10
-        totalPages:
-          type: integer
-          description: Total number of pages
-          example: 10
-        currentPageNumber:
-          type: integer
-          description: Page number of the returned results
-          example: 1
-        currentPageSize:
-          type: integer
-          description: Number of results per page
-          example: 25
     SelfLink:
       properties:
         self:
           type: string
           format: uri
           description: Self-link of current resource
-    PaginationLinks:
-      properties:
-        first:
-          type: string
-          format: uri
-          description: The first page of data
-        last:
-          type: string
-          format: uri
-          description: The last page of data
-        prev:
-          type: string
-          format: uri
-          nullable: true
-          description: The previous page of data
-        next:
-          type: string
-          format: uri
-          nullable: true
-          description: The next page of data
-    PetResource:
+    PositionResource:
       properties:
         id:
           type: string
-          description: A unique ID of pet
+          description: A unique ID of position
+          example: C12345
         type:
           type: string
-          enum:
-            - pet
+          enum: [position]
         links:
           $ref: '#/components/schemas/SelfLink'
         attributes:
           properties:
-            name:
+            title:
               type: string
-              description: Name of pet
-              example: Hedwig
-            owner:
+              description: Position title
+              example: Grad Appt Summer Break
+              nullable: false
+            class:
               type: string
-              description: Name of owner
-              example: Harry Potter
-            species:
+              description: Position class
+              example: Student Project Assistant
+              nullable: false
+            businessCenter:
               type: string
-              description: Species of pet
-              example: Owl
-            age:
-              type: number
-              format: integer
-              description: Age of pet
-              example: 5
-            size:
+              description: Name of the business center
+              example: FOBC
+              nullable: false
+            organization:
               type: object
               properties:
-                length:
-                  type: number
-                  format: float
-                  description: Length in inches
-                  example: 18
-                chest:
+                code:
+                  type: string
+                  nullable: false
+                  description: Organization code for the department
+                  example: '230600'
+                title:
```

Comment:
Should we be consistent with persons and use `description` instead of `title`?

Reply:
Agree.